packages/nextra-theme-docs/src/utils/render-component.tsx

```diff
@@ -1,8 +1,8 @@
-import React from 'react'
+import React, { PropsWithChildren } from 'react'
 
 const renderComponent = <T,>(
-  ComponentOrNode: React.FC<T> | React.ReactNode,
-  props: T,
+  ComponentOrNode: React.FC<PropsWithChildren<T>> | React.ReactNode,
```

Comment:
```suggestion
  ComponentOrNode: React.FC<T> | React.ReactNode,
```
<img width="277" alt="image" src="https://user-images.githubusercontent.com/7361780/180478118-129b6241-f6b1-4c8b-88c3-8ca4ad66f4cb.png">

Reply:
It's for future react 18 update. I will update other FC types later